pyproject.toml

```diff
@@ -18,7 +18,7 @@ dynamic = [
 ]
 dependencies = [
   "asdf>=2.13",
-  "asdf-coordinates-schemas>=0.1",
+  "asdf-coordinates-schemas@ git+https://github.com/WilliamJamieson/asdf-coordinates-schemas.git@add_geodetic", # "asdf-coordinates-schemas>=0.1",
```

Comment:
This should wait for `asdf-coordinates-schemas==0.3.0` to be released. It will have the necessary schema changes.